desc/vmec.py

```diff
@@ -469,7 +469,7 @@ def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
         chi[:] = (
             -2  # negative sign for negative Jacobian
             * Psi
-            * integrate.cumtrapz(r_full * iotaf[:], r_full, initial=0)
+            * integrate.cumulative_trapezoid(r_full * iotaf[:], r_full, initial=0)
```

Comment:
Did they really deprecate this function because of the name? lol

Reply:
yeah there was a whole github issue where they argued about it.